<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DicomExplorer</class>
 <widget class="QMainWindow" name="DicomExplorer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dicom Explorer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTreeView" name="treeView"/>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="styleSheet">
    <string notr="true">QStatusBar::item {
    border: none;
}</string>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="separator"/>
   <addaction name="actionExport"/>
   <addaction name="separator"/>
   <addaction name="actionCollapse"/>
   <addaction name="actionExpand"/>
   <addaction name="actionRefresh"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="DicomExplorer.qrc">
     <normaloff>:/icons/open.svg</normaloff>:/icons/open.svg</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open Dicom File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="icon">
    <iconset resource="DicomExplorer.qrc">
     <normaloff>:/icons/export.ico</normaloff>:/icons/export.ico</iconset>
   </property>
   <property name="text">
    <string>Export</string>
   </property>
   <property name="toolTip">
    <string>Export to text file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionExpand">
   <property name="icon">
    <iconset resource="DicomExplorer.qrc">
     <normaloff>:/icons/expand.svg</normaloff>:/icons/expand.svg</iconset>
   </property>
   <property name="text">
    <string>Expand</string>
   </property>
   <property name="toolTip">
    <string>Expand</string>
   </property>
   <property name="shortcut">
    <string>F15, Ctrl+E</string>
   </property>
  </action>
  <action name="actionCollapse">
   <property name="icon">
    <iconset resource="DicomExplorer.qrc">
     <normaloff>:/icons/collapse.svg</normaloff>:/icons/collapse.svg</iconset>
   </property>
   <property name="text">
    <string>Collapse</string>
   </property>
   <property name="toolTip">
    <string>Collapse</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="icon">
    <iconset resource="DicomExplorer.qrc">
     <normaloff>:/icons/refresh.svg</normaloff>:/icons/refresh.svg</iconset>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
   <property name="toolTip">
    <string>Refresh</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="DicomExplorer.qrc"/>
 </resources>
 <connections/>
</ui>
